airflow/providers/amazon/aws/operators/eks.py

```diff
@@ -26,7 +26,15 @@
 from airflow import AirflowException
 from airflow.models import BaseOperator
 from airflow.providers.amazon.aws.hooks.eks import EksHook
-from airflow.providers.cncf.kubernetes.operators.kubernetes_pod import KubernetesPodOperator
+
+try:
+    from airflow.providers.cncf.kubernetes.operators.kubernetes_pod import KubernetesPodOperator
+except ImportError:
+
+    raise AirflowException(
+        "apache-airflow-providers-cncf-kubernetes not installed, run: "
+        "pip install 'apache-airflow-providers-amazon[cncf.kubernetes]'"
+    )
```

Comment:
Can be made cleaner with `raise ... from None` to suppress the original ImportError